### PR TITLE
Add sign in sign up pages

### PR DESCRIPTION
**Pull request by nottoday2471**
Was created authorization form with link to sign up and recover password
image
Was created sign up form 
image
Was created recover password form
image
Also was added validation for email and password
image
image
image

### CODE REVIEW

File: src/utils/valdiate-password.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export const validatePassword = (password: string) => {
+  const regEx = /((?=.*\d)|(?=.*\W+))(?![.\n])(?=.*[A-Z])(?=.*[a-z]).*$/;
+  return regEx.test(password);
+};
```
Comment (lerik1408) [inline src/utils/valdiate-password.ts:4]: I think a better way is to have one file sharedValidate, for validating shared fields like email, password, uuid.

**Review by lerik1408 — APPROVED** [review]
Good job!

File: src/utils/validate-email.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export const validateEmail = (email: string): boolean => {
```
Comment (BillyAleks) [inline src/utils/validate-email.ts:1]: Yup lib have email validator as well, so you could use it

**Review by BillyAleks — CHANGES_REQUESTED** [review]
Thank you, please check the comments and apply it to all changes that you've done.
I would recommend to use you Yup and Formik to handle and validate forms.
Check autotypes, not everywhere you need an explicit definition.

**Pull request merged**